src/SwitchSelector/SwitchSelector.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { Meta, Story } from "@storybook/react";
```

Comment:
Can we merge these stories into one, use card decorator and write a sentence or two of description?
We dont maintain any technical documentation, so at least keep this one pretty 🙃